src/core/utils/index.js

```diff
@@ -103,7 +103,11 @@ export function sanitizeHTML(text, allowTags = [], _purify = purify) {
 
 // Convert new lines to HTML breaks.
 export function nl2br(text) {
-  return (text || '').replace(/(\r\n|\r|\n)(?!<.+?>)/g, '<br />');
```

Comment:
I don't think any changes should have been required to this function. This will change the behaviour everywhere this is used.